CMakeLists.txt

```diff
@@ -19,9 +19,9 @@ if ("${CMAKE_BUILD_TYPE}" STREQUAL "")
     set(CMAKE_BUILD_TYPE "Debug")
 endif()
 if ("${CMAKE_BUILD_TYPE}" STREQUAL "Release")
-    set(CMAKE_C_FLAGS "-std=gnu99 -pedantic -Wall -Wextra -Wstrict-aliasing -g -O2")
+    set(CMAKE_C_FLAGS "-std=gnu99 -lpam -pedantic -Wall -Wextra -Wstrict-aliasing -g -O2")
```

Comment:
Does including `-lpam` works well with *BSD systems?

Reply:
AFAIK Machinarium does not work with BSD at all.
PAM seems to be Linux only

Reply:
Also, I think this should be removed. Because use `FindPAM` for that.